projects/analytics_logging/resources/variables.tf

```diff
@@ -0,0 +1,14 @@
+variable "bucket_name" {
```

Comment:
Are the `username_base` and `team` variables ever going to be set to anything but these defaults? Why is username_base separate from `bucket_name`?